### ISSUE TITLE
No `tensorflow-rocm` on pypi for Python 3.11

### ISSUE THREAD

**Issue by ivucica** [ROCmSoftwarePlatform/tensorflow-upstream#2146]
### Issue Type

Build/Install

### Have you reproduced the bug with TF nightly?

No

### Source

binary

### Tensorflow Version

n/a, not installable

### Custom Code

Yes

### OS Platform and Distribution

Debian 12 (bookworm)

### Python version

3.11

### Current Behaviour?

I have a Stable Diffusion venv with Python 3.11 and no easy way to downgrade to Python 3.10 on my current Debian stable deployment.

Within venv, running the following errors out:

```
$ pip install tensorflow-rocm
ERROR: Could not find a version that satisfies the requirement tensorflow-rocm (from versions: none)
ERROR: No matching distribution found for tensorflow-rocm
```

Looking at the published `whl`s on http://pypi.org/project/tensorflow-rocm/#files I can see there's the following items:

| Built Distributions | |
| --- | --- |
| tensorflow_rocm-2.11.1.550-cp310-cp310-manylinux2014_x86_64.whl (456.9 MB view hashes) | Uploaded May 10, 2023 cp310 |
| tensorflow_rocm-2.11.1.550-cp39-cp39-manylinux2014_x86_64.whl (456.9 MB view hashes) | Uploaded May 10, 2023 cp39 |
| tensorflow_rocm-2.11.1.550-cp38-cp38-manylinux2014_x86_64.whl (456.8 MB view hashes) | Uploaded May 10, 2023 cp38 |

My current guess is that this is because only 3.8, 3.9 and 3.10 versions are published (and notably there's also no 2.12).

### Standalone code to reproduce the issue

```shell
n/a
```

**Comment by ZappaBoy** [comment]
@ivucica `tensorflow-rocm` is available for `python 3.11` since 10 July 2023 as you can see on [PyPI](http://pypi.org/project/tensorflow-rocm/#files ).

I think that this issue can now be closed.

**Comment by ivucica** [comment]
I’ve been using it for a while now. This can be closed, indeed.
